.eslintignore

```diff
@@ -1,2 +0,0 @@
-node_modules
-tests
```

Comment:
Could we update the `.prettierignore` to include `tests`, and maybe also the other `ignores` items?